documentation/site/docs.css

```diff
@@ -164,11 +164,11 @@ button:focus, .navbar-dark .navbar-toggler:focus {
 .main blockquote {
   font-size: 1.1em;
 }
-@media (min-width: 1050px) {
-  .main p, .main li, .main td, .main th {
-    font-size: 1.2em;
-  }
-  .main blockquote {
+.main li p, .main li li, .main li blockquote {
+  font-size: 1em;
+}
```

Comment:
This feels more like a band-aid than a proper fix, but whatever. I works.

Reply:
I know. I feel dirty writing it. But I have an aversion to `rem` and `px` units, because long ago they were bad for accessibility, but probably aren’t anymore.